includes/definitions/sensatronics-em1.yaml

```diff
@@ -0,0 +1,12 @@
+os: sensatronics-em1
+text: 'Sensatronics EM1'
+type: sensor
+icon: generic
```

Comment:
It is usually a good idea to add an icon.

Reply:
I'll see if I can find an image to create an icon from. Their website is long gone.